perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -117,6 +119,10 @@
 # A global variable initialized once for Azure Blob Service
 _AZURE_BLOB_SERVICE = None
 
+# boto3 recommends creating per-thread sessions. We store them in
+# THREAD_LOCAL.boto_session.
```

Comment:
we need to clarify and confirm the life cycle of a boto3 session here, especially its relationship to the underlying http connection. (i.e., When is a session created and terminated.)

We need to make sure connections are being reused across many requests and we only pay the overhead of connection setup for the first request (per thread)